extensions/wikia/WikiaMaps/css/WikiaMaps.wikiamobile.scss

```diff
@@ -92,7 +92,7 @@ $map-ad-height: 70px;
 
 		a {
 			border-radius: 5px;
-			display: block-inline;
+			display: inline-block;
```

Comment:
I'd double check this one. Browser will likely use `display` property from another selector based on CSS inheritance. So yes, by fixing we can brake things :)